testsuite/forge-cli/src/main.rs

```diff
@@ -635,8 +637,10 @@ fn state_sync_perf_fullnodes_execute_transactions(
     forge_config: ForgeConfig<'static>,
 ) -> ForgeConfig<'static> {
     state_sync_perf_fullnodes_config(forge_config)
-        .with_node_helm_config_fn(Arc::new(|helm_values| {
+        .with_genesis_helm_config_fn(Arc::new(|helm_values| {
```

Comment:
Thanks @rustielin! Do you think separating these out will now be able to override the node config values correctly? e.g., the bootstrapping modes below?

Reply:
Separating the values shouldn't affect anything other than `genesis`. Applying the `genesis` values to `aptos-node` should be a no-op.

However, I'm seeing via the NodeConfig dump on node startup that fullnodes are getting the right bootstrapping mode now. Logs filter for a specific fullnode on this PR (the canary run above)

https://cloud.us.humio.com/k8s/search?query=NodeConfig%0A%7C%20k8s.namespace%20%3D%20%22forge-e2e-pr-4098%22%0A%7C%20%22k8s.pod_name%22%20%3D%20%22aptos-node-2-fullnode-eforge146-0%22&live=false&start=1d

Reply:
Aah, FWIW, the configs you're seeing there are the defaults, so we should actually look at the `forge-state-sync-perf-fullnode-execute` test to see if non-defaults are used 🤔